lib/json.dl

```diff
@@ -34,6 +34,128 @@ typedef JsonValue = // Represents a JSON null value.
                   | // Represents a JSON object.
                     JsonObject{o: Map<string, JsonValue>}
 
+/* Get JsonValue value as `bool`.  Returns `None` if value is not `JsonBool`.
+ */
+function jval_as_bool(v: JsonValue): Option<bool> =
+{
+    match (v) {
+        JsonBool{b} -> Some{b},
+        _ -> None
+    }
+}
+
+/* Get JsonValue value as `bool`.  Returns `def` if value is not `JsonBool`.
+ */
+function jval_as_bool_or(v: JsonValue, def: bool): bool =
```

Comment:
The name is misleading, sounds like boolean or. Should probably be orDefault

Reply:
`or_default` is reserved for functions that return default value of the return type on failure.
I can see that this is slightly confusing, but it is uniform with function naming in this and other libs (and also with the Rust standard library).